README.md

```diff
@@ -69,6 +81,12 @@ if (!map1.has(key)) {
 }
 ```
 
+With this proposal:
+
+```js
+map.upsert(key, () => {}, () => value)
```

Comment:
the update function needs to be undefined or an explicit identity function like v => v